taskcoordinator/app/src/main/java/com/microservices/taskcoordinator/dto/inbound/OrderSubmittedDto.java

```diff
@@ -3,15 +3,23 @@
 import lombok.AllArgsConstructor;
 import lombok.EqualsAndHashCode;
 import lombok.Getter;
+import lombok.NoArgsConstructor;
 import lombok.Setter;
 
+import javax.validation.Valid;
+import javax.validation.constraints.NotNull;
+
 @Getter
 @Setter
 @AllArgsConstructor
 @EqualsAndHashCode
+@NoArgsConstructor
 public class OrderSubmittedDto {
 
+    @NotNull
     private Integer orderId;
 
+    @Valid
```

Comment:
?

Reply:
Annotation in InboundLaundryStateDto would not be checked without that annotation

Reply:
ohh
i see. sorry